### DISCUSSION TITLE
Which package of EMQX Ver4(Open Source) should I install on AmazonLinux2023?

### DISCUSSION THREAD

**Discussion by oshimakunihiko-co** [emqx/emqx#14337] [Q&A] [answered]
When I installed emqx-4.4.19-otp24.3.4.2-1-amzn2-amd64.rpm, the following error occurred and I could not install it.
```
# sudo yum install emqx-4.4.19-otp24.3.4.2-1-amzn2-amd64.rpm -y
Last metadata expiration check: 0:02:49 ago on Wed Dec  4 11:17:03 2024.
Error:
 Problem: conflicting requests
  - nothing provides openssl11 needed by emqx-4.4.19-otp24.3.4.2_1.amzn2.x86_64 from @commandline
(try to add '--skip-broken' to skip uninstallable packages)
```

Please tell me EMQX Ver4 (Open Source) that can be installed on Amazon Linux 2023.

**Comment by thalesmg** [comment, marked as answer]
Hi.  Support for Amazon Linux 2023 in EE 4.4 was added in version 4.4.27.

**Reply by oshimakunihiko-co** [reply]
Thank you, does this mean that EMQX Ver4 for Amazon Linux 2023 in Open Source does not exist?

**Reply by thalesmg** [reply]
Yes, 4.4 community edition stopped being supported and released quite some time ago.

**Reply by oshimakunihiko-co** [reply]
Thank you very much for your reply.
